waspc/packages/deploy/src/providers/railway/README.md

```diff
@@ -0,0 +1,39 @@
+### Take into account things
+
+1. `railway login` does not work in non-interactive mode. Current implementation checks if user logged in, otherwise throw out of the process with a friendly hint what to do next
+2. Global options `--wasp-exe <path>` and `--wasp-project-dir <dir>` are being added to each subcommand in the global scope
+3. This implementation expects a specific file for railway deployments to be present in the main root directory of Wasp - `wasp.deployrc.railway.json`
```

Comment:
Do you think it makes sense to make the `railway.json` file an option, so they can have multiple envs? Similar to how we treat the `fly*.toml` files?

Reply:
yeah absolutely, has to be added